### PR TITLE
Licence, readme and build url

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Vojtech Trefny
```
Comment (vpodzime) [inline LICENCE]: Should actually be ``Red Hat, Inc.``.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  Copr builder
+
+  optional arguments:
+    -h, --help            show this help message and exit
+    -v, --verbose         print debug messages
+    -p [PROJECTS [PROJECTS ...]], --projects [PROJECTS [PROJECTS ...]]
+                          projects to build; if not given, all projects from
+                          config will be built
+    -c CONFIG, --config CONFIG
+                          config file location
```
Comment (vpodzime) [inline README.rst]: Is there any default?

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  ...
+
+
+- **project_name** -- some unique name for project/package
+- **copr_user** -- FAS username or groupname
+- **copr_repo** -- name of the Copr repository (without the username!)
+- **package** -- name of the package that will be built
+- **git_url** -- URL of the Git repo (will be used for "git clone")
+
+  - *spec* file must be in the repo
```
Comment (vpodzime) [inline README.rst]: a *spec* file...

File: README.rst (hunk 2 above)
Comment (vojtechtrefny) [inline README.rst]: No

File: README.rst (hunk 3 above)
Comment (vojtechtrefny) [inline README.rst]: Fixed, thanks.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  copr_user =
+  copr_repo =
+  package =
+  git_url =
+  archive_cmd =
+  git_branch =
+  git_merge_branch =
+
+  [another_project_name]
+  copr_user =
```
Comment (jkonecny12) [inline README.rst]: I think it would be nice to have here example values, like `https://github.com/user/example-project` and similar.

Comment (vojtechtrefny) [inline README.rst]: Fixed, thanks.

**Pull request merged**